addons/test_import_export/models/models_import.py

```diff
@@ -152,3 +152,18 @@ class PathToProperty(models.Model):
     def _compute_all_import_properties(self):
         for record in self:
             record.all_properties_ids = record.properties_id | record.another_properties_id
+
+
+class ImportBaseTranslation(models.Model):
+    _name = _description = 'import.base.translation'
+
+    name = fields.Char(translate=True)
+    description = fields.Char(translate=True)
+    html = fields.Html(translate='html_translate')
```

Comment:
This parameter is actually incorrect (string values are not expected) and only works by accident.